yarn.lock

```diff
@@ -9,10 +9,6 @@ JSONStream@^1.0.0:
     jsonparse "^1.2.0"
     through ">=2.2.7 <3"
 
-abbrev@1:
```

Comment:
Shouldn't we remove this file from the source tree?

Reply:
We should actually keep `.lock` files for reproducibility as it ensures the exact revisions of all dependencies installed (provided we install with `--frozen-lockfile`), no surprises.

Reply:
ok, I'm a yarn noob :rofl: